fvm/evm/stdlib/contract.go

```diff
@@ -97,6 +99,55 @@ func (e abiDecodingError) Error() string {
 	return b.String()
 }
 
+func calculateComputation(
+	inter *interpreter.Interpreter,
+	values *interpreter.ArrayValue,
+	evmAddressTypeID common.TypeID,
+) uint {
+	computation := uint(0)
+
+	values.Iterate(inter, func(element interpreter.Value) (resume bool) {
+		switch value := element.(type) {
+		case *interpreter.StringValue:
+			// Dynamic variables, such as strings, are encoded
+			// in 3 chunks of 32 bytes. The first chunk contains
+			// the index where information for the string begin,
+			// the second chunk contains the number of bytes the
+			// string occupies, and the third chunk contains the
+			// value of the string itself.
+			computation += 2 * abiEncodingByteSize
+			stringLength := len(value.Str)
+			chunks := stringLength / abiEncodingByteSize
+			remainder := stringLength % abiEncodingByteSize
+			if remainder > 0 {
+				chunks += 1
+			}
```

Comment:
```suggestion
  chunks := math.Ceil(stringLen / abiEncodingByteSize)
```

Reply:
Nice one :100: 
Updated in https://github.com/onflow/flow-go/pull/5146/commits/ad809ef779f40d099a69d5da63c3f103a3b06bd5